src/Illuminate/Console/Scheduling/ManagesAttributes.php

```diff
@@ -0,0 +1,204 @@
+<?php
+
+declare(strict_types=1);
```

Comment:
Strict types declaration should be removed (not used in the framework), and could also be a breaking change.

Reply:
Thanks, @stevebauman! I’ve updated the code.